diff -ru netkit-telnet-0.17/telnetd/utility.c netkit-telnet-0.17-patched/telnetd/utility.c
--- netkit-telnet-0.17/telnetd/utility.c	1999-12-12 23:59:45.000000000 +0900
+++ netkit-telnet-0.17-patched/telnetd/utility.c	2017-07-07 23:15:43.055049400 +0900
@@ -306,7 +306,7 @@
 	    if (n > 1) {
 		n = send(net, nbackp, n-1, 0);	/* send URGENT all by itself */
 	    } else {
-		n = send(net, nbackp, n, MSG_OOB);	/* URGENT data */
+		n = send(net, nbackp, n, 0);	/* URGENT data */
 	    }
 	}
     }
